repo_stats/parameters.json

```diff
@@ -1,6 +1,6 @@
 {
     "repo_owner": "sunpy",
-    "repo_name": "sunpy",
+    "repo_names": ["sunpy", "sunpy.org", "package-template", "sunkit-spex", "sunkit-instruments", "drms", "sunkit-magex", "sunpy-sphinx-theme", "ndcube", "sunraster", "streamtracer", "sunkit-image", "sunpy-soar", "sunkit-pyvista", "mpl-animators", "radiospectra", "sunkit-dem", "pyflct", "ablog"],
```

Comment:
So all the code does right now is loop over for stats, it does not take a loop over papers and should be implemented.

Reply:
This will require a slightly larger rewrite first.